src/components/DataRoom/Cexes/Cex.tsx

```diff
@@ -0,0 +1,64 @@
+import css from './styles.module.css'
+import { motion } from 'framer-motion'
+import { Typography } from '@mui/material'
+
+export type CEX = {
+  name: string
+  boxes: number
+  boxColor: string
+  message: string
+}
+
+const textMotionProps = {
+  initial: { opacity: 0, y: -10 },
+  whileInView: { opacity: 1, y: 0 },
+  viewport: { once: true },
+  transition: {
+    type: 'spring',
+    stiffness: 260,
+    damping: 20,
+  },
+}
+
+const gridMotionProps = {
+  initial: { scale: 0, x: -10 },
+  whileInView: { scale: 1, x: 0 },
+  viewport: { once: true },
+  transition: {
+    type: 'spring',
+    stiffness: 260,
+    damping: 20,
+  },
+}
+
+export const Cex = ({ boxes, boxColor, name, message }: CEX) => (
+  <div className={css.cex}>
+    <div className={css.cexGrid}>
+      {[...Array(boxes)].map((_, index) => (
```

Comment:
Nit: to save the spread, we can instantiate the array with the length of `boxes`, e.g.
```suggestion
      {Array.from({ length: boxes },.map((_, index) => (
```